crates/nu-cli/tests/completions/mod.rs

```diff
@@ -299,7 +299,7 @@ fn file_completions() {
     match_suggestions(&expected_paths, &suggestions);
 
     // Test completions for hidden files
-    let target_dir = format!("ls {}{MAIN_SEPARATOR}.", folder(dir.join(".hidden_folder")));
+    let target_dir = format!("ls {}.", folder(dir.join(".hidden_folder")));
```

Comment:
Did this test previously try to complete `ls dir/.hidden_folder//.`? If so, maybe then this could be
```
let target_dir = format!("ls {}", file(dir.join(".hidden_folder").join(".")))
```
?